common/util.go

```diff
@@ -39,6 +39,24 @@ var (
 	ErrFormatProto = fmt.Errorf("unknown VideoProfile format for protobufs")
 	ErrFormatMime  = fmt.Errorf("unknown VideoProfile format for mime type")
 	ErrFormatExt   = fmt.Errorf("unknown VideoProfile format for extension")
+
+	mimeExts = []string{
```

Comment:
Should this just be a `map[string]string` rather than two slices?

Reply:
map will require mutex for syncing, given size of the data - not worth it
actually wanted to add comment about this, but forgot :(
will add one

Reply:
A mutex for what? We're not mutating the table

Reply:
it just stuck in my head that every map access should be guarded...
fixed